### PR TITLE
New @import build process

### PR DESCRIPTION
**Pull request by contolini**
This implements the changes discussed in https://github.com/cfpb/capital-framework/issues/151.
## Additions
- `@import` rules to pull in dependencies.
## Removals
- Grunt bower and concat tasks
## Changes
- `.bowerrc` points to `src/vendor` (the now defunct Grunt task used to do this).
- Updated some JS paths in the test runner.
- Bumped to `1.0.0`.
## Testing
- You can't. :neutral_face: There's a chicken vs. the egg scenario here. This project's dependencies now point to the 1.x.x version of other CF components. But those versions aren't in Bower/npm yet because their PRs are still pending. This is why Travis is failing.
- The `dev` branch was thoroughly tested prior to this PR and more testing will occur afterward. There will likely be some kinks to work out after this merge but that's okay -- semver will shield current projects from any bugs.
## Review
- @Scotchester 
- @anselmbradford

### CODE REVIEW

**Comment by contolini** [comment]
See https://github.com/cfpb/capital-framework/pull/169#issuecomment-107689434

File: test/cf-expandables.html [hunk 1]
```diff
@@ -158,9 +158,9 @@
 
     <!-- </div> -->
 
-    <script src="../src/vendor/jquery/jquery.js"></script>
-    <script src="../src/vendor/jquery.easing/jquery.easing.js"></script>
-    <script src="../src/vendor/qunit/qunit.js"></script>
+    <script src="../src/vendor/jquery/dist/jquery.js"></script>
```
Comment (anselmbradford) [inline test/cf-expandables.html:161]: Do you care if this is minified or not?

Comment (contolini) [inline test/cf-expandables.html:161]: Mmm, I don't care. Should I care?

Comment (anselmbradford) [inline test/cf-expandables.html:161]: It's probably fine but `jquery.min.js` would be the production environment while `jquery.js` would be the dev environment, so in case there was some kind of crazy edge case where minification created an issue with cf-expandables it would show up here.
